lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetWriter.java

```diff
@@ -240,6 +241,7 @@ static Slice getFooter(List<RowGroup> rowGroups, MessageType messageType)
     {
         FileMetaData fileMetaData = new FileMetaData();
         fileMetaData.setVersion(1);
+        fileMetaData.setCreated_by(FULL_VERSION);
```

Comment:
We don't use parquet-mr in the trino-parquet writer. This is incorrect and will mislead other writers.

We should add our own identifier here AFAIK.

Reply:
`parquet-mr` still gets used as a library. The experimental writer is "native" in the fact that it invokes the `parquet-mr` library directly instead of using the Hive writer. 

The `parquet-mr` library expects a particular version to be parsable from that string (see: https://github.com/apache/parquet-mr/blob/master/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetRecordReader.java#L194).

Reply:
Ah, it'll imported from our repackaged trino-hive-apache. I was looking for org.apache.parquet itself and couldn't find references.

Today I learnt.